crosswalks/publiccode.csv

```diff
@@ -0,0 +1,74 @@
+Property,publiccode
+codeRepository,url
+programmingLanguage,
+review,
+runtimePlatform,platforms
```

Comment:
https://docs.italia.it/italia/developers-italia/publiccodeyml-en/en/master/schema.core.html#key-platforms says:

> This key specifies which platform the software runs on. It is meant to describe the platforms that users will use to access and operate the software, rather than the platform the software itself runs on.

this seems to explicitly exclude mapping to runtimePlatform.

Reply:
Ok! This description is in fact very confusing to me because it seems to say 2 contradictory things (`the platform the software runs on` and `not the platform the software runs on`. :smile:

Reply:
Hah, indeed. I missed the first sentence was the exact opposite of the second one...